landoui/revisions.py

```diff
@@ -20,7 +21,25 @@ def get_revision(revision_id):
     revision_api_url = '{}/revisions/{}'.format(
         os.getenv('LANDO_API_URL'), revision_id
     )
-    result = requests.get(revision_api_url)
+
+    try:
+        result = requests.get(revision_api_url)
+        result.raise_for_status()
+    except requests.HTTPError as exc:
+        if exc.response.status_code == 404:
+            # The user looked up a non-existent revision, no special treatment
+            # is necessary.
+            abort(404)
+        else:
+            sentry.captureException()
+            abort(502)
+    except requests.ConnectTimeout:
```

Comment:
This and the following exception blocks aren't needed. The UI will only display generic 500 errors, and then sentry will pass on the detailed exception to us. The above else statement should take care of this.